src/libostree/ostree-repo-libarchive.c

```diff
@@ -244,7 +244,7 @@ aic_get_final_path (OstreeRepoArchiveImportContext *ctx, const char *path, GErro
       char *ret = ctx->opts->translate_pathname (ctx->repo, &stbuf, path,
                                                  ctx->opts->translate_pathname_user_data);
       if (ret)
-        return ret;
+        return g_strdup (ret);
```

Comment:
Won't this just introduce a leak? `translate_pathname` is defined as returning a `char *` that the caller takes ownership of, so I think this was right. Maybe coverity is confused by the conditional part?